consensus/src/liveness/proposal_generator.rs

```diff
@@ -317,14 +318,19 @@ impl ProposalGenerator {
             let voting_power_ratio = proposer_election.get_voting_power_participation_ratio(round);
 
             let (
-                max_block_unique_txns,
+                max_block_txns_after_filtering,
```

Comment:
@vusirikala I've changed this from unique to after_filtering, as with your other change this will be target after all filtering (both unique and expired)

Reply:
Cool. I've used `max_unique_txns` in a bunch of places. I will change the naming everywhere.